Updates for mc RELEASE.2025-02-04T04-57-50Z (#1427)
minio/docs: source/operations/concepts/thresholds.rst

This list is not exhaustive and may not apply to your operating system and filesystem combination.

> On Unix-like operating systems, objects where the path name is just ``.``, ``..``, or ``/`` return an error of ``file access denied``.

Reviewer: maybe
```suggestion
On Unix-like operating systems, objects with a path name of ``.``, ``..``, or ``/`` return an error of ``file access denied``.
```

Consult your operating system vendor or filesystem documentation for a comprehensive list for your situation.

> MinIO recommends using Linux operating system with an XFS based filesystem for production workloads.

Author: Going with the first one.
